pkg/cmd/server/api/v1/types.go

```diff
@@ -696,3 +697,128 @@ type AssetExtensionsConfig struct {
 	// Web Console's context root. Defaults to false.
 	HTML5Mode bool `json:"html5Mode"`
 }
+
+type LDAPSyncConfig struct {
```

Comment:
@liggitt This type isn't registered externally, so you can't use it yet.  I'd rather not argue about it in this pull.  It needs reworking, but lets rework it elsewhere.